### PR TITLE
Admin from terminal

### LINKED ISSUE #21: [FEAT] :sparkles: Create admin user from command line
### Description

We can't create a rest api endpoint for creating admin user because if we do than anyone can use that endpoint to create
admin user for them

We should implement a command that will run in terminal which will create a new user in database with role admin

example command:-

```bash
admin -e <email> -p <password>
```

### Anything else?

I have already implemented this in another project you can get reference of code from here:-
https://github.com/shriproperty/server/blob/dev/bin/index.ts
https://github.com/shriproperty/server/blob/dev/package.json#L12-L14

### PR DESCRIPTION
**Pull request by BitOfAByte**
### Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Checklist:

- [x] This pull request follow our contribution guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (postman and swagger)
- [ ] I have written tests for the code
- [ ] I have updated .env.example file for new .env variables

### What is current behavior?

Closes #(issue_number)

### CODE REVIEW

File: src/utils/admin.terminal.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  .usage('admin -e <email> -u <username> -p <password>')
+  .options({
+    email: { type: 'string', demandOption: true, alias: 'e' },
+    username: { type: 'string', demandOption: true, alias: 'u' },
+    password: { type: 'string', demandOption: true, alias: 'p' },
+  })
+  .parseSync();
+
+mongoose.connect(process.env.DB_URI as string, () => {
+  UserModel.create({
```
Comment (aayushchugh) [inline src/utils/admin.terminal.ts]: add new property while creating the user

verified: true

**Review by aayushchugh — CHANGES_REQUESTED** [review]
Add new property in package.json `bin` check code here
https://github.com/shriproperty/server/blob/dev/package.json#L12-L14

and move the file from utils to a new `bin` folder in root of the directory
and add `#!/usr/bin/env node` on top of the file so that npm can figure out that it is 
a command that will be used in terminal

more reference: -
https://github.com/shriproperty/server/blob/dev/bin/index.ts

**Review by aayushchugh — APPROVED** [review]
nice

**Review by is-it-ayush — APPROVED** [review]
Hey Ayush! The .gitignore has a lot of useless ignores we won't need as of this project. Later we can refactor it. I'm approving it for now.

**Comment by aayushchugh** [comment]
@is-it-ayush I just used vscode extension to create it 🙂

**Comment by is-it-ayush** [comment]
@aayushchugh Understandable. We can definately refactor it later in the Optimization Phase to reduce the load size even if it's minute. For now, It's all good.

**Pull request merged**